docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-role-authorization-proc.adoc

```diff
@@ -1,2 +1,46 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-role-authorization-proc-{context}']
+= Role authorization
+
+Role assignments can be given when the case starts or while it is active at any time, and can be changed at any point while the case instance is active. Case Management users should always assign roles to task assignments rather than actual user or group names. This allows the case to be as dynamic as possible, as actual user or group assignment can happen as late as possible, and can also be changed.
+
+Roles are authorized to perform certain case management tasks when starting a new case instance using the Showcase application or the REST API. 
+
+Use the following procedure to start a new IT Orders case using the REST API.
+
+.Prerequisite
+
+* The IT Orders sample project has been imported in {CENTRAL} and deployed to the {KIE_SERVER}.
+
+
+.Procedure
+
+. Create a `POST` REST API call with the following endpoint:
++
+`http://host:port/kie-server/services/rest/server/containers/itorders/cases/itorders.orderhardware/instances`
++
+* `itorders`: the container alias that has been deployed to the {KIE_SERVER}.
+* `itorders.orderhardware`: the name of the case definition.
+
+. Provide the following role configuration in the request body:
++
+[source,java]
+----
+{
+  "case-data" : {  },
+  "case-user-assignments" : {
+    "owner" : "maciek",
+    "manager" : "maciek"
```

Comment:
I'd use different actors for owner and manager just for better readability

Reply:
Changed to "cam" as that's the case manager persona.